### PR TITLE
add gpu driver 560.35.03

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: tests/scripts/.definitions.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ TERRAFORM="terraform -chdir=${TERRAFORM_DIR}"
 : ${LOG_DIR:="/tmp/logs"}
 : ${PROJECT:="$(basename "${PROJECT_DIR}")"}
 : ${TEST_NAMESPACE:="test-operator"}
-: ${TARGET_DRIVER_VERSION:="535.183.06"}
+: ${TARGET_DRIVER_VERSION:="560.35.03"}
```
Comment (cdesiniotis) [inline tests/scripts/.definitions.sh:20]: Question: Do we want to always test the default driver version in the operator helm chart?

Comment (tariq1890) [reply tests/scripts/.definitions.sh:20]: This is the target driver version used for the driver upgrade tests in our e2e suite

Comment (tariq1890) [reply tests/scripts/.definitions.sh:20]: The default driver version for the e2e tests is taken from the values.yaml which is R550 currently

Comment (cdesiniotis) [reply tests/scripts/.definitions.sh:20]: Ah, got it.

**Review by cdesiniotis — APPROVED** [review]

**Pull request merged**